src/WebJobs.Script.WebHost/Models/FunctionAppSecrets.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Microsoft.Azure.WebJobs.Script.WebHost.Management;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost.Models
+{
+    /// <summary>
+    /// Secrets cache format used by both <see cref="FunctionsSyncManager"/> and <see cref="StartupContextProvider"/>.
+    /// </summary>
+    public class FunctionAppSecrets
```

Comment:
Now that we're both serializing (for SyncTriggers) and deserializing (for startup context) this payload, it makes sense to have a Type for it to prevent mistakes.